src/bytedeck_summernote/widgets.py

```diff
@@ -0,0 +1,93 @@
+import bleach
+import html as htmllib
+import json
+
+from django.conf import settings as django_settings
+from django.forms.utils import flatatt
+from django.template.loader import render_to_string
+from django.urls import reverse
+from django.utils.safestring import mark_safe
+
+from bleach.css_sanitizer import CSSSanitizer
+from django_summernote.utils import get_config
+from django_summernote.widgets import SummernoteInplaceWidget, SummernoteWidget
+
+
+class ByteDeckSummernoteAdvancedWidget(SummernoteWidget):
+    def render(self, name, value, attrs=None, **kwargs):
+        """Override default `render` method to use customized template file"""
+        summernote_settings = self.summernote_settings()
+        summernote_settings.update(self.attrs.get("summernote", {}))
+
+        # disable XSS protection for CodeView (mandatory for ByteDeck project)
+        summernote_settings.update(
+            {
+                "codeviewIframeFilter": False,  # set this to false to skip filtering entities.
+            }
+        )
+
+        html = super(SummernoteWidget, self).render(name, value, attrs=attrs, **kwargs)
+        context = {
+            "id": attrs["id"],
+            "id_safe": attrs["id"].replace("-", "_"),
+            "flat_attrs": flatatt(self.final_attr(attrs)),
+            "settings": json.dumps(summernote_settings),
+            # using bytedeck's url here...
+            "src": reverse("bytedeck_summernote-editor", kwargs={"id": attrs["id"]}),
+            # Width and height have to be pulled out to create an iframe with correct size
+            "width": summernote_settings["width"],
+            "height": summernote_settings["height"],
+        }
+
+        # using customized template here...
+        html += render_to_string("bytedeck_summernote/widget_advanced.html", context)
+        return mark_safe(html)
+
+
+class ByteDeckSummernoteSafeWidget(SummernoteInplaceWidget):
```

Comment:
Docstring needed for this class describing the purpose of the SafeWidget (what does it add/change from it's base class?)